app/forms/idv/api_image_upload_form.rb

```diff
@@ -94,9 +102,45 @@ def validate_form
       )
 
       analytics.idv_doc_auth_submitted_image_upload_form(**response)
+      track_upload_attempt(response)
+
       response
     end
 
+    def track_upload_attempt(response)
+      return unless attempts_api_tracker.enabled?
+
+      # back_metadata = write_image(image: back.present? ? back_image_bytes : nil)
+      back_metadata = write_image(image: readable?(:back) ? back_image_bytes : nil)
```

Comment:
Do we want a separate config for whether writing files is enabled? I'm thinking of the sandbox case where we don't have S3 set up, but we also don't necessarily want to write the files to disk.

Reply:
i was not sure! i think it's probably good to have a separate config. separately, will we be able to set it up in sandbox, so that we can manually accept the doc escrow working? (and so we can work on developing a script to return the files)

Reply:
I think it would be good to separate, yeah. I can make a ticket for and work on the infrastructure pieces.

Reply:
added here, that made some of the tests cleaner too https://github.com/18F/identity-idp/pull/12117/commits/7eea1b83019b688ab5afeb368215768b8565931d

Reply:
ticket: https://gitlab.login.gov/lg-teams/Melba/protocols-backlog/-/issues/164